sp800-63-3/sec5_xAL.md

```diff
@@ -31,14 +33,14 @@ A summary of each of the identity, authenticator, and federation assurance level
 
 | Identity Assurance Level |
 |:----------------------|
-| **IAL1** - At this level, attributes provided in conjunction with the authentication process, if any, are self-asserted.|
+| **IAL1** - At this level, attributes, if any, are self-asserted or should be treated as self-asserted.|
 | **IAL2** - IAL 2 introduces the need for either remote or in-person identity proofing. IAL 2 requires identifying attributes to have been verified in person or remotely using, at a minimum, the procedures given in [SP 800-63A](sp800-63a.html).|
 | **IAL3** - At IAL 3, in-person identity proofing is required. Identifying attributes must be verified by an authorized representative of the CSP through examination of physical documentation as described in [SP 800-63A](sp800-63a.html).|
 
 |Authenticator Assurance Level|
 |:----------------------|
-|**AAL1** - AAL 1 provides some assurance that the claimant controls an authenticator registered to a subscriber. AAL 1 uses single-factor authentication using a wide range of available authentication technologies. Successful authentication requires that the claimant prove through a secure authentication protocol that he or she possesses and controls an authenticator.|
-| **AAL2** – AAL 2 provides high confidence that the claimant controls authenticator(s) registered to a subscriber. Two different authentication factors are required. Approved cryptographic techniques are required at AAL 2 and above.|
+|**AAL1** - AAL 1 provides some assurance that the claimant controls an authenticator registered to a subscriber. AAL 1 uses single-factor authentication using a wide range of available authentication technologies. Successful authentication requires that the claimant prove possession and control of the authenticator(s) through a secure authentication protocol.|
```

Comment:
same.  I think we are going to standardize on no space between xAL and #.  Deloitte can do that.